packages/server/lib/browsers/chrome.ts

```diff
@@ -318,8 +317,124 @@ const _handleDownloads = async function (client, dir, automation) {
   })
 }
 
-const _setAutomation = async (client: CRIWrapper.Client, automation: Automation, closeCurrentTarget: () => Promise<void>) => {
-  const cdpAutomation = await CdpAutomation.create(client.send, client.on, closeCurrentTarget, automation)
+let frameTree
+let gettingFrameTree
+
+const onReconnect = (client: CRIWrapper.Client) => {
```

Comment:
not exactly a 1:1 here, but this callback handler is defined here and passed into [browserCRIClient static constructor](https://github.com/cypress-io/cypress/pull/21237/files#diff-b5b66d85423fa714add2e866448d3411b0f1cdf2cb4b4a95aa80394fa3edd1b3R87) which is ultimately referenced in the generic [criClient static contructor](https://github.com/cypress-io/cypress/pull/21237/files#diff-4eee46e0819a421b758964abfcf2ea357773afceaff00ceafd16b89f193f4c4eR107). [original](https://github.com/cypress-io/cypress/pull/18075/files#diff-e2888f195a1c64929f7e4de7cb14eba872a682eb1cd29f8d50b87131c1f78c1aR261) in `9.x`